src/nsmgr/internal/pkg/deviceplugin/utils.go

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) 2020 Cisco and/or its affiliates.
```

Comment:
It seems we could probably eliminate this entire file and increase the clarity of the code overall by simply computing these items as variables in Allocate() and also using path.Join.